docker-compose.yml

```diff
@@ -26,15 +32,19 @@ services:
       - target: 5000
         published: 5000
         protocol: tcp
-        mode: host
-    entrypoint: poetry run python /buildingmotif/buildingmotif/api/app.py
+        mode: host 
+    entrypoint: poetry run /opt/start.sh
 
   app:
     container_name: buildingmotif-app
     depends_on: # require that the api server has started
       - api
     build: ./buildingmotif-app
     image: buildingmotif-app:latest
+    volumes:
+      - type: bind
+        source: ./buildingmotif-app
+        target: /buildingmotif-app
```

Comment:
Is this for debugging purposes? In the future we may want to get rid of this mount or move it to a compose file which adds debugging capabilities to the base compose file.

Reply:
I think this also makes it harder to build images that we could distribute, if we wanted. I'll comment it out for now with a note saying it can be uncommented to facilitate debugging